index.html

```diff
@@ -1642,17 +1642,12 @@ <h4>DIDCommMessaging</h4>
       "https://www.w3.org/ns/did/v1",
       "https://didcomm.org/messaging/contexts/v2"
   ],
-  ...
   "type":"DIDCommMessaging",
-  "serviceEndpoint":"http://example.com/path",
-  "accept":[
-      "didcomm/v2",
-      "didcomm/aip2;env=rfc587"
-  ],
-  "routingKeys":[
-      "did:example:somemediator#somekey"
-  ]
-  ...
+  "serviceEndpoint": {
```

Comment:
[Current DIDComm V2 spec](https://identity.foundation/didcomm-messaging/spec/v2.0/#service-endpoint) says that `serviceEndpoint` _MUST contain an ordered list of objects._. This makes me think that, if you have a single endpoint, you'll need to define it as a list containing this single object.

However, in this example (and also [Peer DID method specification](https://identity.foundation/peer-did-method-spec/#multi-key-creation)), `serviceEndpoint` is encoded as an object.

Reply:
I'm sorry, I was looking at DIDComm v2.0 but the latest stable version is [v2.1](https://identity.foundation/didcomm-messaging/spec/v2.1/), which allows for the encoding of an object. That explains why all other specs were 'wrong' :D